index.html

```diff
@@ -0,0 +1,643 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.11.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+    <title>The New Web Replica</title>
+</head>
+<body>
+   <!-- <nav class="top-nav">
+        <section class="topNav-left">
+            <a href="#">New</a>
+            <a href="#">Events</a>
+            <a href="#">Business</a>
+            <a href="#">AMAs</a>
+            <a href="#">Space</a>
+            <a href="#">Terms & Conditions</a>
+        </section>
+        <section class="topNav-right">
+            <a href="#">About</a>
+            <a href="#">Advertise</a>
+            <a href="#">Jobs</a>
+            <a href="#">Conact</a>
+        </section>
+    </nav> -->
```

Comment:
- [ ] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** `/* .nav { color: red;} */`  
:heavy_check_mark: **comments about code** `/* Navigation */`